.lgtm.yml

```diff
@@ -19,6 +19,3 @@ path_classifiers:
     - ".dependabot"
     - ".github"
     - ".lgtm.yml"
-
-generated:
```

Comment:
@aaron-mcgrath-adp  -> this was exactly the wrong thing to do : c.f. https://quotidian-ennui.github.io/blog/2019/10/28/lombok-lgtm/

Since you are using lombok you should follow the blog post to make sure we get our code scanned; and add "generated" to your .gitnignore.